src/Parser/Priest/Discipline/Modules/Spells/TouchOfTheGrave.js

```diff
@@ -28,32 +29,45 @@ class TouchOfTheGrave extends Analyzer {
     }
   }
   on_byPlayer_heal(event) {
+    const spellId = event.ability.guid;
+    if (spellId === SPELLS.TOUCH_OF_THE_GRAVE.id) {
+      this.directHealing += event.amount + (event.absorbed || 0);
+    }
+
     if (!isAtonement(event)) {
       return;
     }
     if (this.atonementSource.atonementDamageSource) {
       if (this.atonementSource.atonementDamageSource.ability.guid !== SPELLS.TOUCH_OF_THE_GRAVE.id) {
         return;
       }
-      this.healing += event.amount + (event.absorbed || 0);
+      this.atonementHealing += event.amount + (event.absorbed || 0);
     }
   }
 
   statistic() {
-    const healing = this.healing || 0;
+    const atonementHealing = this.atonementHealing || 0;
+    const directHealing = this.directHealing || 0;
     const damage = this.damage || 0;
-    
-    // since we can't directly get undead racial status, if we found 0 damage, 
+
+    const atonementPct = this.owner.getPercentageOfTotalHealingDone(atonementHealing);
+    const directPct = this.owner.getPercentageOfTotalHealingDone(directHealing);
+    const totalPct = atonementPct + directPct;
+
+    // since we can't directly get undead racial status, if we found 0 damage,
     // assume they aren't undead and do not load the module
     if(damage === 0) { return; }
-
+    console.log(this.directHealing);
```

Comment:
You left this in here. While you're at it could you change the above return statement to return `null` instead? It's preferred to either always return nothing (void) or always return a value, but not switch between the two.